app/src/main/java/ar/com/wolox/android/example/ui/example/home/HomeFragment.kt

```diff
@@ -0,0 +1,42 @@
+package ar.com.wolox.android.example.ui.example.home
+
+import ar.com.wolox.android.R
+import ar.com.wolox.wolmo.core.adapter.viewpager.SimpleFragmentPagerAdapter
+import ar.com.wolox.wolmo.core.fragment.WolmoFragment
+import kotlinx.android.synthetic.main.fragment_home.*
+import javax.inject.Inject
+
+/**
+ *
+ */
+class HomeFragment : WolmoFragment<HomePresenter>(), IHomeView {
+
+    @Inject internal lateinit var homeNewsFragment: HomeNewsFragment
+    @Inject internal lateinit var homeProfileFragment: HomeProfileFragment
+
+    private lateinit var fragmentPagerAdapter: SimpleFragmentPagerAdapter
+
+    companion object {
+
+        enum class TabIndex(val index: Int) {
+
+            NEWS(0),
+            PROFILE(1)
+        }
+    }
+
+    override fun init() {
+        fragmentPagerAdapter = SimpleFragmentPagerAdapter(childFragmentManager)
+        fragmentPagerAdapter.apply {
+            addFragment(homeNewsFragment, getString(R.string.fragment_home_news_tab_title))
+            addFragment(homeProfileFragment, getString(R.string.fragment_home_profile_tab_title))
+        }
+
+        vHomeViewPager.adapter = fragmentPagerAdapter
+        vHomeTab.setupWithViewPager(vHomeViewPager)
+        vHomeTab.getTabAt(TabIndex.NEWS.index)?.setIcon(R.drawable.home_tab_news_icon_selector)
```

Comment:
It's necessary to use index? If using TabIndex.NEWS it will give a 0 due to the fact that you created NEWS(0)

Reply:
Yes, I thought the same, but apparently is necessary.

![image](https://user-images.githubusercontent.com/48364172/55744159-a5d6c780-5a0a-11e9-9a04-003f0f0386e7.png)